tests/snmpsim/ceraos.snmprec

```diff
@@ -0,0 +1 @@
+1.3.6.1.2.1.1.2.0|6|1.3.6.1.4.1.2281.1.20.2.2
```

Comment:
Yes please leave the snmprec file as you see from an actual device, it's to test the discovery code and shouldn't be based on exactly what discovery is looking for but what is actually on a device :)